### DISCUSSION TITLE
How can I set SOCIALACCOUNT_ONLY from allauth

### DISCUSSION THREAD

**Discussion by DonDebonair** [TandoorRecipes/recipes#3246] [Q&A]
`django-allauth` has a setting called `SOCIALACCOUNT_ONLY` that let's you remove the local login and force users to login using a social account. I want to use this so all logins happen through Authelia, which I use as an OIDC provider.

How can I set this variable for Tandoor? My assumption is that setting it as an environment variable won't work, as `recipes/settings.py` does not explicitly reference that environment variable. What other way can I set it then?

**Comment by martin53757** [comment]
I also wonder how I can only allow login via OIDC
